### DISCUSSION TITLE
Linux Version

### DISCUSSION THREAD

**Discussion by GreenVenom77** [Tyrrrz/YoutubeDownloader#433] [Q&A] [answered]
After the project has been ported to Avalonia is there any consideration to deploy the project to linux as I really want to try it
and I have read this blog here as they have enabled it in Avalonia

https://avaloniaui.net/Blog/avalonia-the-premier-choice-for-net-developers-targeting-linux

**Comment by Tyrrrz** [comment, marked as answer]
Yes, I'm planning to do that. At the moment, I'm in the process of migrating my other project (https://github.com/Tyrrrz/DiscordChatExporter) to Avalonia — I will try to make it cross-platform first (since it's the easiest to experiment on) and once that works, I'll apply those learnings/changes to this project as well.

**Reply by GreenVenom77** [reply]
Thank you for the answer and I really appreciate your work on the project.
